frontend/src/components/NotFound/styles/Button.ts

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+const ButtonContainer = styled.button`
```

Comment:
Let's use antd components like button, card, etc. 
https://ant.design/components/

Reply:
Hi @nidhi-tandon but if we see the Figma design ant button components would not reflect it

Reply:
Check out their default button. https://ant.design/components/button/#API
Possible to use this with updated styles matching our Figma styles. antd will provide support for the light theme as well and other states of the button as well (hover, active, etc)

Reply:
Hi @nidhi-tandon 

I have updated the ButtonContainer with the Link as suggested by @jyash97 

But I encounter another problem when we are using the styled component with the antd component. The styles are having higher specificity than the styles written in the styled components 

Below the reference for same

<img width="850" alt="Screenshot 2021-07-17 at 3 52 26 PM" src="https://user-images.githubusercontent.com/29013861/126033940-97f65628-8cf2-4ea3-bdc7-03c850f5b86f.png">